crashreports/internal/win32/CrashReportContext.cpp

```diff
@@ -0,0 +1,115 @@
+#include <locale>
+#include <codecvt>
+#include <sstream>
+#include "client/windows/handler/exception_handler.h"
+#include "CrashReportContext.h"
+
+namespace {
+	bool uploadReport(
+		const wchar_t* dump_path,
+		const wchar_t* minidump_id,
+		void* context,
+		EXCEPTION_POINTERS* exinfo,
```

Comment:
It helps to avoid compiler warnings about unused parameters.  Either omit the variable name, or comment it out, or inside the function write
`
exinfo;
assertion;
`